app/marketing/views.py

```diff
@@ -630,7 +639,7 @@ def leaderboard(request, key=''):
         ranks = LeaderboardRank.objects.filter(active=True, leaderboard=key)
 
     amount = ranks.values_list('amount').annotate(Max('amount')).order_by('-amount')
-    items = ranks.order_by('-amount')
+    items = ranks.order_by(f'-{order_by}')
```

Comment:
This is going to modify the assumed functionality of `order_by` - If a dev is familiar with how order_by functions, i.e.: `-amount` returning in descending order versus `amount` being ascending, this will force all ordering to `descending` and break when `-` is passed.